template.html

```diff
@@ -350,10 +352,10 @@ <h2><a href="#others">
           </div>
         </div>
       </section>
-    </div>
+    </main>
 
     <footer>
-      <a href="index.html">English</a>
+      <a href="/">English</a>
```

Comment:
Is this a change from accessibility perspective, or something that was slipstreamed into this PR? Either way, PR #59 completely changes the language switching so it's probably best to take this change out of this PR and either make it part of #59 or do it in a separate follow-up?

Reply:
The content is marked as duplicate without his change.
If this is also fixed in #59 I can remove this?

Reply:
> The content is marked as duplicate without his change.

Hmm? Not sure what that means? Who or what marks it as "duplicate"? The line only appears once, right?

Either way, you can check [the change at the other PR that'll conflict](https://github.com/infi-nl/the-infi-way/pull/59/files#diff-fafe98a5a4bafa0bee5631b9949eb8257151368637c18eeb6c37deca4aa391f0) so I recommend reverting this one change here, possibly suggesting an improvement to the other PR if there's still something to be fixed?